package.json

```diff
@@ -25,7 +25,8 @@
   },
   "homepage": "https://github.com/FormidableLabs/webpack-dashboard#readme",
   "peerDependencies": {
-    "webpack": "*"
+    "webpack": "*",
+    "webpack-dev-server": "*"
```

Comment:
Is this supposed to be a `peerDependency`? We don't `require` `webpack-dev-server` anywhere in the code...